### PR TITLE
Feature/cli interface

### PR DESCRIPTION
**Pull request by ky1ejs**
MVP for printing code coverage data. Gives others something to start working off and contribute with.

### CODE REVIEW

File: Parasol/Coverage.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        let task = NSTask()
+        task.launchPath = "/usr/bin/xcrun"
+        task.arguments = ["llvm-cov", "show", "-instr-profile", profdataPath, executablePath]
+        task.launch()
+        task.waitUntilExit()
+    }
+    
+    static func showCoverageReport(profdataPath: String, executablePath: String) -> Coverage {
+        let task = NSTask()
+        task.launchPath = "/usr/bin/xcrun"
```
Comment (khasinski) [inline Parasol/Coverage.swift]: Hardcoded, extract config?

File: Parasol/main.swift [hunk 2]
```diff
@@ -8,5 +8,39 @@
...
+            print("Coverage data has not been generated. Would you like to run tests? (y/n)")
+            let input = readLine()
+            if input == "y" {
+                target.runTests()
+                if case let .Exists(profdataPath, executablePath) = target.coverageDataExists {
+                    for line in Coverage.showCoverageReport(profdataPath, executablePath: executablePath).lines {
+                        print(line)
+                    }
+                } else {
+                    print("ERROR: Coverage data was not generated by Xcode.")
```
Comment (khasinski) [inline Parasol/main.swift]: Print to stderr?

Comment (ky1ejs) [inline Parasol/main.swift]: You mean this should be printing the error to standard error?

Comment (khasinski) [inline Parasol/main.swift]: Yes :) Every process is initialised with three open file descriptors, stdin , stdout , and stderr. Errors should go to stderr so you can extract them in logs.

Comment (ky1ejs) [inline Parasol/main.swift]: Cool, will fix

**Pull request merged**